sentry-android-core/src/test/java/io/sentry/android/core/InternalSentrySdkTest.kt

```diff
@@ -101,6 +105,61 @@ class InternalSentrySdkTest {
 
             InternalSentrySdk.captureEnvelope(data)
         }
+
+        fun mockFinishedAppStart() {
+            val metrics = AppStartMetrics.getInstance()
+
+            metrics.appStartType = AppStartMetrics.AppStartType.WARM
+
+            metrics.appStartTimeSpan.setStartedAt(20) // Can't be 0, as that's the default value if not set
```

Comment:
Yes, I always though `0` was a good idea (rather than making it nullable), but actually it's painful for writing tests 🙈